_data/simple-icons.json

```
@@ -7510,6 +7510,11 @@
                 ]
             }
         },
+        {
+            "title": "React Query",
+            "hex": "FF0000",
+            "source": "https://github.com/tannerlinsley/react-query/blob/master/docs/src/images/emblem-light.svg"
```

As per [the contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines) the source should include the commit hash of the commit from which the SVG came.

```suggestion
            "source": "https://github.com/tannerlinsley/react-query/blob/76e314d0887edbd702369e8554cc840508561b94/docs/src/images/emblem-light.svg"
```